packages/turf-buffer/index.js

```diff
@@ -94,16 +94,24 @@ function buffer(geojson, radius, units, steps) {
     }
 
     // Project GeoJSON to Transverse Mercator projection (convert to Meters)
-    var distance = radiansToDistance(distanceToRadians(radius, units), 'meters');
-    var projection = defineProjection(geojson);
-    var projected = {
-        type: geometry.type,
-        coordinates: projectCoords(geometry.coordinates, projection)
-    };
+    var projected;
+    var bbox = turfBbox(geojson);
+    var needsTransverseMercator = bbox[1] > 50 && bbox[3] > 50;
+
+    if (needsTransverseMercator) {
+        var projection = defineProjection(geometry);
```

Comment:
@stebogit Did you want to move over this `defineProjection` to `@turf/projection` (as `define`).

```js
/**
 * Define Transverse Mercator projection
 *
 * @private
 * @param {Geometry|Feature<any>} geojson Base projection on center of GeoJSON
 * @returns {GeoProjection} D3 Geo Transverse Mercator Projection
 */
function defineProjection(geojson) {
    var coords = center(geojson).geometry.coordinates.reverse();
    var rotate = coords.map(function (coord) { return -coord; });
    var projection = d3.geoTransverseMercator()
        .center(coords)
        .rotate(rotate)
        .scale(6373000);

    return projection;
}
```

Reply:
@DenisCarriere wouldn't it be even more useful having a `@turf/projection.toTransverse` method for the actual conversion? I mean, why would you define the (Transverse in this case) projection but for converting the coordinates?
 
In that regard, at this point we might need to re-think the structure of the `projection` module to leave it open to accept maybe other projections in the future, if ever needed, like:
- `@turf/projection.toMercator`
- `@turf/projection.fromMercator`
- `@turf/projection.toTransverse`
- `@turf/projection.fromTransverse`
- ...

knowing the base projection for Turf is `WGS84`